lib/msal-angular/docs/v2-docs/configuration.md

```diff
@@ -29,6 +29,8 @@ Please see our [MsalInterceptor](https://github.com/AzureAD/microsoft-authentica
 
 We recommend importing `MsalRedirectComponent` and bootstrapping with the `AppComponent` if you intend to use redirects. Please see the [redirect documentation](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular/docs/v2-docs/redirects.md) for more details. 
 
+**Note:** As of MSAL v3.x, initialization of the application object is now required. See the [v2-v3 upgrade guide](../v2-v3-upgrade-guide.md) for more details.
```

Comment:
This doc is in v2-docs folder. Do we need to call out v3 things here?

Reply:
Since there aren't many changes between v2 and v3, I will move the v2 docs into the main docs folder, so they can apply to both v2 and v3. v1 docs will remain in their own folder.

Reply:
Actually, since there are external links to the v2 folder, I won't move the docs. Instead, I will call out in the v3 migration doc that most of the v2 docs apply. If there are many more changes, I will create specific v3 docs per topic.